packages/appeals-service-api/src/services/back-office-v2/formatters/utils.js

```diff
@@ -146,7 +150,9 @@ exports.formatApplicationSubmissionUsers = ({
 			firstName: appellantFirstName,
 			lastName: appellantLastName,
 			emailAddress: null,
-			serviceUserType: 'Appellant'
+			serviceUserType: 'Appellant',
+			telephoneNumber: appellantPhoneNumber,
```

Comment:
I think this field is named wrong in submission, should be contactPhoneNumber, it's asked to the person filling out the appeal, not asking for appellant

Can we rename it in submission table, fieldName in questions and then add to just the primary contact

Reply:
Yeah agreed, this did feel like weird handling